src/main/java/yesman/epicfight/api/client/camera/EpicFightCameraAPI.java

```diff
@@ -355,6 +352,16 @@ public boolean setNextLockOnTarget(int direction) {
         return next.isPresent();
     }
 
+    /**
+     * Creates a compact projection matrix without view, hurt bob
+     */
```

Comment:
Nit: Consider using markdown comments as we're migrating from HTML comments:

```suggestion
     /// Creates a compact projection matrix without view, hurt bob
```